docs/guides/getting-started.md

```diff
@@ -0,0 +1,78 @@
+# Getting started
+
+## Introduction
+
+The Medrunner API allows you to perform all the operations that you do with our web client. It is built using REST principles which ensures predictable URLs that makes writing applications easy. This API follows HTTP rules, enabling a wide range of HTTP clients which can be used to interact with the API. Every resource is exposed as a URL.
+
+The base URL for the API is:
+
+```
+https://api.medrunner.space/
+```
+
+## TypeScript Client
+
+To make it easier to interact with the Medrunner API, we provide a TypeScript client that you can use in your projects. The client is available on npm and can be installed using the following command:
+
+```bash
+npm i @medrunner/api-client
+```
+
+Once installed, the client will provide you with methods to call API endpoints. Each method will provide you the type of the body and the response.
+The client will also handle authentication for you, so you don't have to worry about it.
+
+Here is an example of how to configure the client:
+
+```ts
+import MedrunnerApiClient from "@medrunner/api-client";
+
+const apiConfig = {
+  baseUrl: "https://api.medrunner.space",
+  token: "YOUR_REFRESH_TOKEN",
+};
+
+const refreshCallback = async newTokens => {
+  localStorage.setItem("refreshToken", newTokens.refreshToken);
+};
```

Comment:
again, unnecessary

Reply:
Removed mentions of the refresh callback in 1289ab2e23a0b185962533363eaa925ef6f50cb1